lib/krikri/engine.rb

```diff
@@ -51,5 +51,10 @@ class Engine < ::Rails::Engine
       end
     end
 
+    initializer :register_harvesters do
+      Krikri::Harvester::Registry
+        .register('OAI', Krikri::Harvesters::OAIHarvester)
```

Comment:
Should this be `Krikri::Harvester::Registry.(:oai, Krikri::Harvesters::OAIHarvester)`? (See my comment [here](https://github.com/dpla/KriKri/pull/46/files#r22553261).)